### ISSUE TITLE
Missing LEFT command before REMOVEUSER

### ISSUE THREAD

**Issue by p2004a** [beyond-all-reason/teiserver#195]
Teiserver seems to incorrectly send the `REMOVEUSER` command without sending `LEFT` command.

This causes the spads to generate a lot of logs like

```
20231110191519 - WARNING  - [SpringLobbyInterface] Missing LEFT command before REMOVEUSER (user "Jazcash", channel: main)
```

that just pollute the output when looking at logs.

**Comment by Teifion** [comment]
I have hotfixed this in, please let me know if I can close the issue @p2004a

**Comment by p2004a** [comment]
I've just checked recent logs and it's definitely better, but not resolved:

```
20231119085914 - WARNING  - [SpringLobbyInterface] Missing LEFT command before REMOVEUSER (user "[teh]clusterUS[00]", channels: main, autohosts, spads)
20231119085929 - WARNING  - [SpringLobbyInterface] Missing LEFT command before REMOVEUSER (user "[teh]cluster1[17]", channels: main, autohosts, spads)
20231119085937 - WARNING  - [SpringLobbyInterface] Missing LEFT command before REMOVEUSER (user "mutlifishies", channel: main)
```

Almost all at the moment are related to the cluster bot users, with only a very occasional single user showing up.

**Comment by L-e-x-o-n** [comment]
Is this stil lan issue?

**Comment by p2004a** [comment]
Yes
```
debian@spads-eu1:~$ cat /home/spads/spads/var/ClusterManager/Host\[EU1\]\[001\]/log/spads.log | grep REMOVEUSER | tail -n 1
20240822213436 - WARNING  - [SpringLobbyInterface] Missing LEFT command before REMOVEUSER (user "Dr_Pebble", channel: main)
```

**Comment by p2004a** [comment]
What's more, it's 60% of all lines present in the SPADS logs.

**Comment by L-e-x-o-n** [comment]
Fixed by #479 . Thanks @bigbluejay9 !
